packages/forms/src/__tests__/form.test.js

```diff
@@ -68,6 +71,18 @@ describe('Form', () => {
     )
   }
 
+  const TestComponentWithRef = () => {
+    const inputEl = React.useRef(null)
+    React.useEffect(() => {
+      inputEl.current.focus()
+    })
```

Comment:
a shot in the dark, would it help to pass an empty array as the second argument to useEffect()?

Reply:
That would make sure it only runs once when the component is mounted, which makes sense here!